docs/_documentations/troubleshooting.md

```diff
@@ -120,6 +121,43 @@ This error occurs because the custom plug-in repository link contains an additio
 **Workaround**
 Remove the extra space from the custom plug-in repository link. 
 
+***
+# Upgrading Codewind
+
+<!--
+Action/Topic: Upgrading Codewind
+Issue type: bug/info
+Issue links: https://github.com/eclipse/codewind/issues/2604
+0.11.0: New for this release
+-->
+
+## Error upgrading Codewind to the latest
+When upgrading Codewind from 0.4, 0.5 or 0.8.1 to the latest version, you see the following error: 
```

Comment:
instead of listing the versions, it could be 'from any version older than 0.9 to the latest'

Reply:
Corrected